playbook-website/config/webpack/environment.js

```diff
@@ -13,6 +13,16 @@ environment.loaders.insert('javascript', {
   include: path.resolve(__dirname, '../../app'),
 })
 
+environment.loaders.insert('svgr', {
+  test: /\.(svg)$/,
+  use: {
+    loader: '@svgr/webpack',
+  },
+  include: [
+    path.resolve(__dirname, '../../../../node_modules/@powerhome/playbook-icons/icons')
+  ],
+})
```

Comment:
Website only config that just says: "load SVGs from this directory as React elements"

Reply:
😍